README.md

```diff
@@ -51,12 +55,15 @@ Simply copy the **menas.war** file produced when building the project into Tomca
 #### Standardization and Conformance requirements:
 - **Spark 2.4.0** installation
 - **Hadoop 2.7** installation
+- **Menas** running instance
 - **Menas Credentials File** in your home directory (a configuration file for authenticating the Spark jobs with Menas) 
 e.g. `~/menas-credential.properties`:
 ```
 username=user
 password=changeme
 ```
+ - **_INFO file** must be present in the **RAW** dataset directory (this is a file tracking control measures via [Atum](https://github.com/AbsaOSS/atum), example can be found [here](examples/data/input/_INFO))
+ - **Directory structure** for the **RAW** data should follow the convention of `<path_to_dataset_in_menas>/<year>/<month>/<day>/v<dataset_version>`
```

Comment:
the date represented must be the same as in the `report-date` option in standartization